mama/c_cpp/src/c/bridge/qpid/codec.c

```diff
@@ -212,7 +213,7 @@ qpidBridgeMsgCodec_unpack (msgBridge        bridgeMessage,
     /* If this looks like another MAMA payload type */
     else if (PN_BINARY == firstAtom.type)
     {
-        mamaPayloadType     payloadType     = MAMA_PAYLOAD_UNKNOWN;
+        char payloadType =  '\0';
```

Comment:
Should use MAMA_PAYLOAD_NULL or MAMA_PAYLOAD_ID_NULL or whatever you call the macro literal as opposed to the enumerator type.